docs/en/integrations/data-ingestion/clickpipes/index.md

```diff
@@ -114,10 +102,10 @@ The supported formats are:
 
 | Format                                                                                    | Support     |
 |-------------------------------------------------------------------------------------------|-------------|
-| [JSON](../../../interfaces/formats.md/#json)                                               | ✔           | 
+| [JSON](../../../interfaces/formats.md/#json)                                               | ✔           |
 | [AvroConfluent](../../../interfaces/formats.md/#data-format-avro-confluent)                |*Coming Soon*|
-| [TabSeparated](../../../interfaces/formats.md/#tabseparated)                               |*Coming Soon*| 
-| [CSV](../../../interfaces/formats.md/#csv)                                                 |*Coming Soon*| 
+| [TabSeparated](../../../interfaces/formats.md/#tabseparated)                               |*Coming Soon*|
+| [CSV](../../../interfaces/formats.md/#csv)                                                 |*Coming Soon*|
 
 ## Supported data types
```

Comment:
I think this might need to be forked for cases where standard JSON is not the source? For S3, we'll support anything clickhouse itself supports. I'm guessing Avro gets more complicated cc @genzgd. 
(Sorry, outside the scope of this PR but relevant to how this page becomes more generic).